src/redux/Books/Books.js

```diff
@@ -1,33 +1,11 @@
-const ADD_BOOK = 'bookStore/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/vFAUZGx5x4z5msHtqnDH/books';
+
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const FETCH_BOOK = 'bookstore/books/FETCH_BOOK';
 
```

Comment:
- [ ] Good Job so far :+1:  But as mentioned in the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md) for this project, " Use` fetch `or `axios` with `createAsyncThunk` for making API calls. ", please make sure that createAsyncThunk is used in your project for creating thunk action creators in order to make your project works fine :smile: . You can check the example below :arrow_down: 
```
export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 

```